src/Concordium/Client/GRPC2.hs

```diff
@@ -0,0 +1,3222 @@
+{-# LANGUAGE DataKinds #-}
```

Comment:
If we are copying this from the node we should do that in a separate PR so that we can also update the node, and have a clear separation so we don't introduce issues by accident.